torch/_inductor/codegen/wrapper.py

```diff
@@ -1194,31 +1200,34 @@ def codegen_tensor_option(self, device, dtype):
         cpp_device = self.codegen_device(device)
         return f"at::TensorOptions({cpp_device}).dtype({DTYPE_TO_ATEN[dtype]}))"
 
+    def codegen_allocation(self, buffer):
```

Comment:
This was introduced to address another subtle issue. `AllocateLine` may insert  a `ReuseLine` when running its planning. So instead of inserting `AllocateLine` here, we can just write out the outputs assignment.